tests/test_cli.py

```diff
@@ -238,6 +238,44 @@ def test_setting(args):
     assert settings["default_page_size"] == 5
 
 
+def test_plugin_s_overwrite():
+    runner = CliRunner()
+    plugins_dir = str(pathlib.Path(__file__).parent / "plugins")
+
+    result = runner.invoke(
+        cli,
+        [
+            "--plugins-dir",
+            plugins_dir,
+            "--get",
+            "/_memory.json?sql=select+prepare_connection_args()",
+        ],
+    )
+    assert result.exit_code == 0, result.output
+    assert (
+        json.loads(result.output).get("rows")[0].get("prepare_connection_args()")
```

Comment:
I couldn't remember how this worked, found the answer here:

https://github.com/simonw/datasette/blob/a4c96d01b27ce7cd06662a024da3547132a7c412/tests/plugins/my_plugin.py#L14-L27